web/src/containers/Summary/_Summary.scss

```diff
@@ -160,7 +160,68 @@ $trade-tab--arrow-size: 0.4rem;
 		opacity: 0.5;
 	}
 }
-
+.ant-select-single:not(.ant-select-customize-input) .ant-select-selector {
+	background-color: #202020;
```

Comment:
We should not hard-code colors. We already have different colors for this. Also, this is not theme-specific.